### PR TITLE
Update Trace documentation and samples.

### PR DESCRIPTION
**Pull request by iantalarico**

### CODE REVIEW

File: apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Snippets/AspNetSnippets.cs [hunk 1]
```diff
@@ -29,5 +32,27 @@ public static void Register(HttpConfiguration config)
...
+            void Application_Start(object sender, EventArgs e)
+            {
+                string projectId = "[Google Cloud Platform project ID]";
+                // Trace a sampling of incoming Http requests.
+                CloudTrace.Initialize(projectId, this);
+            }
+        }
+        // End sample
+
+        // Sample: UseTracer
```
Comment (jskeet) [inline apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Snippets/AspNetSnippets.cs]: Is it important that this sample includes the method declaration itself? Most of our samples don't - it makes sense for InitializeTrace for context, but I'm not sure it does in this case.

File: apis/Google.Devtools.AspNet/docs/index.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 `Google.Devtools.AspNet` is an ASP.NET instrumentation library for Google Stackdriver.
 It allows for simple integration of Stackdriver into ASP.NET applications with minimal code changes.
 
-`Google.Devtools.AspNet` currently supports Stackdriver Error Reporting.
+`Google.Devtools.AspNet` currently supports Stackdriver Error Reporting and Stackdriver Tracing.
```
Comment (jskeet) [inline apis/Google.Devtools.AspNet/docs/index.md]: Tracing or Trace? https://cloud.google.com/trace/ says Trace, but maybe there's a branding change afoot?

File: apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Snippets/AspNetSnippets.cs (hunk 1 above)
Comment (iantalarico) [inline apis/Google.Devtools.AspNet/Google.Devtools.AspNet.Snippets/AspNetSnippets.cs]: No, not required. Updated.

File: apis/Google.Devtools.AspNet/docs/index.md (hunk 2 above)
Comment (iantalarico) [inline apis/Google.Devtools.AspNet/docs/index.md]: Should be 'Trace', fixed.

**Pull request merged**